pytorch_to_returnn/torch/_C.py

```diff
@@ -149,6 +149,10 @@ def originating_tensor_axis(self) -> int:
     return tensor_entry.torch_axis_from_returnn_axis[returnn_axis]
 
   def as_tensor(self):
+    if self.merged_dims and self.originating_tensor is None:
+      from functools import reduce
+      from operator import mul
+      return reduce(mul, [dim.as_tensor() for dim in self.merged_dims], 1)
```

Comment:
`as_tensor()` doesn't work (or might not work?) in case of static dims? Not sure.

Reply:
For static dims, we could also return `int(self)`.